plugins/modules/pnp_intent.py

```diff
@@ -517,6 +530,7 @@ def get_reset_params(self):
             ]
         }
 
+        self.log("Paramters used for resetting from deleted state are {0}".format(str(reset_params)), "INFO")
```

Comment:
self.log("Parameters used for resetting from the deleted state: {0}".format(str(reset_params)), "INFO")

Reply:
Resolved